create_model.py

```diff
@@ -11,21 +11,27 @@
 
 from matplotlib import pyplot as plt
 
+# REMEMBER TO CHANGE THESE
+MODELNAME = 'ralf/b200-e8'
+PATH_TO_DATA = './sketches/'
+
+BATCHES = 200
```

Comment:
Should this be named equally to the input name as `BATCH_SIZE`?

Reply:
> Should this be named equally to the input name as `BATCH_SIZE`?

It could, but it doesn't really matter